storybook/src/docs/developer-guide/web-app.docs.mdx

```diff
@@ -1,30 +1,30 @@
 {/* @license CC0-1.0 */}
 
 import { Meta } from "@storybook/blocks";
-import { WebAppIcons } from "./components/AppIcons";
+import { WebAppIcons } from "../components/AppIcons";
 
-<Meta title="Docs/Assets/Web App" />
+<Meta title="Docs/Developer Guide/Web App" />
 
-# Progressive Web App: manifest and icons
+# Web App
 
-Turn a website into a Progressive Web App (PWA) with a Web Manifest and app icons.
+Turn a website into a Progressive Web App (PWA) with a web manifest and app icons.
 
 Here’s how to publish a web app with the proper app names and icons.
 This set covers all combinations of common operating systems (Android, iOS, macOS, and Windows) and browsers (Chrome, Safari, Edge, and Firefox).
 
-> There is seperate documentation on [how to add a website icon (a “favicon”)](/docs/docs-assets-web-app--docs).
+> There is seperate documentation on [how to add a website icon (a “favicon”)](/docs/foundation-assets-favicon--docs).
 
-## Usage
+## How to Use
```

Comment:
We don’t do either very consistently. Storybook uses title case in their sidebar. I think we started applying that to the Docs headings of these stories to not introduce a difference there. Then it would make sense to have title case everywhere.

On the other hand, sentence case is easier to read an maybe more conventional in The Netherlands. I can go along with sentence case in headings everywhere while letting Storybook do what it does (or find out if we can configure those as well).

Choose the latter?

Reply:
Yeah, sounds like a good approach